core/src/components/list-header/list-header.ionic.scss

```diff
@@ -4,10 +4,12 @@
 
 :host {
   --background: transparent;
+  --color: #{globals.$ionic-color-neutral-1200};
 
   min-height: 58px;
 
   font-size: globals.$ionic-font-size-350;
+  font-weight: globals.$ionic-font-weight-medium;
```

Comment:
Should this use `body/md/medium` instead?

Reply:
Good catch! https://github.com/ionic-team/ionic-framework/pull/29907/commits/668fe1b6ee331ae863f604caf5ce23d7f17e9f15